app/helpers/application_helper/toolbar/middleware_servers_center.rb

```diff
@@ -18,6 +18,16 @@ class ApplicationHelper::Toolbar::MiddlewareServersCenter < ApplicationHelper::T
           :send_checked => true,
           :enabled      => false,
           :onwhen       => "1+"),
+        button(
+          :middleware_server_check_compliance,
+          'fa fa-search fa-lg',
+          N_('Check Compliance of the last known configuration for these Middleware Servers'),
+          N_('Check Compliance of Last Known Configuration'),
+          :url_parms    => "main_div",
+          :send_checked => true,
+          :confirm      => N_("Initiate Check Compliance of the last known configuration for the selected items?"),
+          :enabled      => "false",
```

Comment:
what is the influence of enabled=false here?

Reply:
By default this button is disabled, unless more than 1 item is selected. I know that the whole group is disabled unless one item is enabled, but felt right to make it disabled and enable it after at least one item is selected.